lib/ImageWorker.js

```diff
@@ -3,15 +3,10 @@
 import React, { Component } from 'react';
 
 const webWorkerScript = `
+  const handleResponse = () => response.blob();
```

Comment:
response is not defined here. When this fn is executed it will break you can change it to 
```javascript
response => response.blob()
```

Reply:
fixed